src/sys/personality.rs

```diff
@@ -80,10 +80,9 @@ pub fn get() -> Result<Persona> {
 ///
 /// Example:
 ///
-// Disable test on aarch64 until we know why it fails.
```

Comment:
This failure seemed to be due to changes on Google's side.  But they reverted it after a few days, so you can revert this part, too.

Reply:
Done